PlaybackLab/DownloadToOwnSampleApp/README.md

```diff
@@ -20,21 +20,19 @@ To run this app, we assume you have an Apple developer account because you need
 * **OOOoyalaPlayer**: The player we always provide to playback video.
 * **OOOoyalaPlayerViewController**: Used to manage the `view` of the player.
 
 On top of these classes, we built the provided sample app that you can find here. It uses all the functionality described in the objects above. Here are the important classes you want to check out:
-
-* **AssetPersistenceManager**: This is a singleton object which manages multiple `OOAssetDownloadManager` objects. It means you can check this example to manage multiple downloads at the same time. This class is the `OOAssetDownloadManagerDelegate` for all the `OOAssetDownloadManager` objects, so you'll get notified here on the progress of a download. `AssetPersistenceManager` broadcasts notifications in the app so anyone interested can add itself as an observer of these notifications and act accordingly as the downloads progress.
 * **AssetListViewController**: Manages the `UITableView` showing the assets of the app. Depending on the download state of each asset, it will show different options when tapping the accessory button of each cell. Options can be _Download_, _Cancel_, or _Delete_.
-* **OptionTableViewCell**: An instance of each cell rendered in the `UITableView`. Each cell is an observer of the download notifications sent by `AssetPersistenceManager`, so the cell refreshes the UI when there's a new download state.
+* **OptionTableViewCell**: An instance of each cell rendered in the `UITableView`. Each cell passes it's block with task to appropriate instance of `OODtoAsset` to be notified when download state changes, so the cell refreshes the UI.
 * **OptionsDataSource**: Defines the video assets in the app. This is the data that each cell in the `UITableView` will show, and the configuration the download and player objects will use.
 * **PlayerSelectionOption**: Is the object containing all the information of a particular asset.
 * **BasicEmbedTokenGenerator**: An example implementation of `OOEmbedTokenGenerator`. Some specifics should not be performed this way in a production app, because it is using an Ooyala's account API_KEY and API_SECRET that we discourage using this way. Everything done here using API_KEY and API_SECRET information should be performed securely in your own server and your app should be calling that server instead.
-* **PlayerViewController**: Uses an `OOOoyalaPlayerViewController` to show a player view and playback the video. Here's where online or offline playback can happen. This object is an observer of the `AssetPersistenceManager` notifications, so it knows when an asset has been downloaded and can be played offline.
+* **PlayerViewController**: Uses an `OOOoyalaPlayerViewController` to show a player view and playback the video. Here's where online or offline playback can happen. This object should pass block with task to linked `OODtoAsset` to be notified when download state changes and know when an asset has been downloaded and can be played offline.
```

Comment:
Please, look carefully at the `PlayerViewController` class. It has a property of `OODtoAsset` type, we don't have to pass any "blocks"

Reply:
I've changed description of `PlayerViewController`